app/shared/directives/noMobileSupport/noMobileSupport.css

```diff
@@ -0,0 +1,16 @@
+.no-mobile-support {
+	display: none;
```

Comment:
Perhaps position: absolute, top: 0, left: 0, width: 100%, height: 100% is the way to go here?